src/datasource/utils.ts

```diff
@@ -39,7 +39,7 @@ export function useDatasourceFields(datasource: BaseQuickwitDataSource, range: T
     const wordIsField = word.match(/([^:\s]+):"?([^"\s]*)"?/);
     if (wordIsField?.length) {
       const [_match, fieldName, _fieldValue] = wordIsField;
-      const candidateValues = await datasource.getTagValues({ key: fieldName, timeRange: range });
+      const candidateValues = await datasource.getTagValues({ key: fieldName, timeRange: range, fieldValue: _fieldValue});
```

Comment:
you should remove the underscore in `_fieldValue`
variables starting with underscore means they are not used.